src/app/grid-toolbar/grid-toolbar-custom.sample.ts

```diff
@@ -9,6 +9,9 @@ import { IgxColumnComponent } from 'igniteui-angular';
 })
 export class GridToolbarCustomSampleComponent {
 
+    @ViewChild('grid1', { read: IgxGridComponent })
+    public igxGrid1: IgxGridComponent;
```

Comment:
As mentioned before, no need for "igxGrid1".

Reply:
Addressed.